pages/bookshelf.tsx

```diff
@@ -0,0 +1,53 @@
+import { NextPage } from "next"
+import "react-toastify/dist/ReactToastify.css"
+import Navigation from "../components/bookshelf/Navigation"
+import { Container } from "../components/styled/bookshelf"
+import { useState, useContext, useEffect, useMemo } from "react"
+import Discover from "../components/bookshelf/discover/Discover"
+import LogoutBox from "../components/bookshelf/LogoutBox"
+import { UserContext } from "../context/UserContext"
+import { useRouter } from "next/router"
+import { getBooks } from "../api/books"
+import { BooksContext } from "../context/BooksContext"
+
+const Home: NextPage = () => {
+  const [active, setActive] = useState("Reading List")
```

Comment:
Try using enums (that can be shared through export in another file) for this one so there's a single source of truth for your state.

sample
```
enum ActivePageEnum {
 reading_list = "Reading List"
 something_else = "Something Else"
}

const [active, setActive] = useState<ActivePageEnum | string>(ActivePageEnum.reading_list)

### conditional

active == ActivePageEnum.reading_list ? true_response : false_response
```

Reply:
![image](https://user-images.githubusercontent.com/91857581/188588080-6b42536f-e730-4271-987b-1ee7af2e662c.png)